### PR TITLE
TEC-6427 : create linkedIn api follower statistics objects

### PR DESCRIPTION
**Pull request by clementEchobox**
### Description of Changes
Added POJOs related to the follower statistics and added test

### Documentation
N/A

### Risks & Impacts
Low risk at the moment since not in use

### Testing
Used the unit test to make sure the pojos are correctly deserialized

### Compare (For layered PRs)
https://github.com/clementEchobox/ebx-linkedin-sdk/compare/TEC-4766-CreateLinkedInEngagementPOJO...clementEchobox:TEC-6427-CreateLinkedinAPIFollowerStatisticsObjects

### CODE REVIEW

File: src/test/java/com/echobox/api/linkedin/types/statistics/StatisticsTest.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  /**
+   * test that an statistics JSON is correctly deserialized
+   * json taken from : https://developer.linkedin.com/docs/guide/v2/organizations/
+   * follower-statistics
+   * 
+   */
+  @Test
+  public void testEngagementJson() {
+    
+    String json = "{\n" + "  \"paging\": {\n" + "    \"count\": 10,\n" + "    \"start\": 0\n"
```
Comment (am6010) [inline src/test/java/com/echobox/api/linkedin/types/statistics/StatisticsTest.java]: I think is better to export in a json file and import it like in `DefaultJsonMapperTest`

File: src/main/java/com/echobox/api/linkedin/types/statistics/FollowerCount.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+/**
+ * @author clementcaylux on 03/12/2018.
+ */
+public class FollowerCount {
+  
+  @Getter
+  @Setter
+  @LinkedIn
+  private long organicFollowerCount;
```
Comment (am6010) [inline src/main/java/com/echobox/api/linkedin/types/statistics/FollowerCount.java]: Maybe a Long ? If for some reason is the value is null

File: src/main/java/com/echobox/api/linkedin/types/Share.java [hunk 3]
```diff
@@ -24,9 +24,9 @@
 import lombok.Setter;
 
 /**
- * Share POJO
+ * ShareStatistics POJO
```
Comment (eddspencer) [inline src/main/java/com/echobox/api/linkedin/types/Share.java]: Have been over-zealous on the find and replace? This is still the `Share` class, please check all comment changes that have changes `Share` -> `ShareStatistics` as many of these look wrong.

File: src/main/java/com/echobox/api/linkedin/types/objectype/Locale.java [hunk 4]
```diff
@@ -24,7 +24,7 @@
 /**
  * Locale POJO
  * @see <a href="https://developer.linkedin.com/docs/ref/v2/object-types#Locale">
- * Share Text</a>
+ * ShareStatistics Text</a>
```
Comment (eddspencer) [inline src/main/java/com/echobox/api/linkedin/types/objectype/Locale.java]: This should say `Locale` anyway

**Pull request merged**